### DISCUSSION TITLE
Storage for Actions and Packages limit reached with self-hosted runner

### DISCUSSION THREAD

**Discussion by akoskm** [community/community#25988] [Actions]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/81bf0a4a8d476f211e26af526cbaa889bc2f5a66.png" data-download-href="/uploads/short-url/ivMTw1Mj4BGs8pM97Yq03lzoTsO.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3111627/181094441-51692858-31de-4454-ae94-721a1935162c.png" alt="image" data-base62-sha1="ivMTw1Mj4BGs8pM97Yq03lzoTsO" width="690" height="109" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/81bf0a4a8d476f211e26af526cbaa889bc2f5a66_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">761×121 12.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/07676085d59b764fcc74074c6d65233f0a498cc6.png" data-download-href="/uploads/short-url/13uPfodXWdik3R2ovPmUcUGhzWm.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3111627/181094442-e8887c0f-51ec-4d37-be44-7189bcf6d6d7.png" alt="image" data-base62-sha1="13uPfodXWdik3R2ovPmUcUGhzWm" width="690" height="242" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/07676085d59b764fcc74074c6d65233f0a498cc6_2_690x242.png, https://aws1.discourse-cdn.com/github/original/2X/0/07676085d59b764fcc74074c6d65233f0a498cc6.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/07676085d59b764fcc74074c6d65233f0a498cc6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/07676085d59b764fcc74074c6d65233f0a498cc6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">751×264 17 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I was under the impression that until I use self-hosted runners I’m good to go and can work without a build limit etc.</p>
<p>I’m not sure I understand why GitHub wants to charge us.</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/akoskm">@akoskm</a>,</p>
<p>It’s free to use self-hosted runner with GitHub Actions, will not increase the actions minutes for billing.</p>
<p>However, it’s possible to incur a fee for extra data storage(eg: packages, artifacts) and data transfer above the default limit. More details please refer to the official doc <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions" rel="nofollow noopener">here </a>.</p>
<p>Thanks.</p>

**Comment by akoskm** [comment, marked as answer]
<p>My bad, I just realized we used the <code>actions/upload-artifact@v2</code> to upload these artifacts that we never actually used:</p>
<pre><code class="lang-auto">    - name: Upload server bundle
      uses: actions/upload-artifact@v2
      with:
        name: dist-server
        path: dist-server/

    - name: Upload client bundle
      uses: actions/upload-artifact@v2
      with:
        name: dist
        path: dist/
</code></pre>
<p>I think this ate our storage package. :man_facepalming:</p>
